recipes/snax/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "snax" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://software.igwn.org/sources/source/{{ name }}-{{ version }}.tar.gz
+  sha256: 5db5c3c1a88e25cda69f2808d9987f607ba252f73edb6c5f526b471e25df3165
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=42
+    - setuptools-scm >=3.4
+    - wheel
+    - pip
+  run:
+    - gstreamer
+    - gst-plugins-base
+    - gstlal >=1.10.0
+    - gstlal-ugly >=1.10.0
+    - h5py
+    - jinja2
+    - ligo-scald >=0.8
+    - ligo-segments
+    - numpy
+    - pluggy
+    - python >=3.6
+    - python-lal
+    - python-ligo-lw
+    - pygobject
+    - scipy
+
+test:
+  imports:
+    - snax
+  commands:
+    - export TMPDIR=$(python -c "import tempfile; print(tempfile.gettempdir())")
```

Comment:
Is this needed?

Reply:
Thanks @ocefpaf, I had thought I did due to an upstream package requiring it on import, but turns out I don't. I have removed this export.